src/main/java/com/google/sps/data/GovPostReq.java

```diff
@@ -17,9 +17,9 @@
 /** Adaptor class for the government data source.
  * The adaptors get their information from their respective data sources - the government API is called here. 
  */
-public class GovAdaptor {
+public class GovPostReq {
   
-  public static ArrayList<NSWGovAQDataPoint> extract() throws Exception {
+  public static ArrayList<NSWGovAQDataPoint> extractAQI() throws Exception {
```

Comment:
What is the reasoning for making extractAQI static?

Reply:
Because this was from the file GovPostReq, which was previously called GovAdaptor. It had a public static void that tested how the function ran, and  that only works when extractAQI was static. But that isn't the case here, so i'm getting rid of it here.